src/components/tables/PoolsTable/PoolsTable.vue

```diff
@@ -330,12 +332,29 @@ function iconAddresses(pool: Pool) {
               :pickedTokens="selectedTokens"
             />
           </div>
-          <BalChip
-            v-if="isLiquidityBootstrapping(pool.poolType)"
-            label="LBP"
-            color="amber"
-          />
-          <BalChipNew v-else-if="pool?.isNew" class="mt-1" />
+          <div>
```

Comment:
Optional change: we could extract this into a new `<PoolToolTip :pool="pool"/>`  where we could unit test all the combinations.